database/migration_test.go

```diff
@@ -24,7 +26,7 @@ func (s *migrationSuite) TestMigrationSuccess(c *gc.C) {
 
 	db := s.DB()
 	m := NewDBMigration(db, stubLogger{}, delta)
-	c.Assert(m.Apply(), jc.ErrorIsNil)
```

Comment:
`context.Background` in the tests. We know we're not going to try threading something else here later.